packages/frontend-2/components/dashboard/Sidebar.vue

```diff
@@ -233,4 +237,13 @@ const openWorkspaceCreateDialog = () => {
     source: 'sidebar'
   })
 }
+
+onWorkspaceResult((result) => {
```

Comment:
When users open the app we fetch all workspaces to add them to the sidebar, this hooks into that